src/overlays/actors/ovl_En_Stone_heishi/z_en_stone_heishi.h

```diff
@@ -2,18 +2,68 @@
 #define Z_EN_STONE_HEISHI_H
 
 #include "global.h"
+#include "assets/objects/object_sdn/object_sdn.h"
 
 struct EnStoneheishi;
 
 typedef void (*EnStoneheishiActionFunc)(struct EnStoneheishi*, PlayState*);
 
 typedef struct EnStoneheishi {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x110];
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[SOLDIER_LIMB_MAX];
+    /* 0x1EE */ Vec3s morphTable[SOLDIER_LIMB_MAX];
     /* 0x254 */ EnStoneheishiActionFunc actionFunc;
-    /* 0x258 */ char unk_258[0x6C];
+    /* 0x258 */ s16 headRotY;
+    /* 0x25A */ s16 headRotX;
+    /* 0x25C */ s16 headRotZ;
+    /* 0x25E */ s16 targetHeadPosY;
+    /* 0x260 */ s16 targetHeadPosX;
+    /* 0x264 */ s32 animIndex;
+    /* 0x268 */ s16 timer;
+    /* 0x26A */ s16 DrinkBottleState;
+    /* 0x26C */ f32 endFrame;
+    /* 0x270 */ s16 action;
+    /* 0x272 */ s16 textIndex;
+    /* 0x274 */ s16 textIdSet;
+    /* 0x276 */ u8 bottleDisplay;
+    /* 0x277 */ u8 playerGivesBluePotion;
+    /* 0x278 */ ColliderCylinder collider;
 } EnStoneheishi; // size = 0x2C4
 
+typedef enum EN_STONE_ACTION {
+    /* 0 */ EN_STONE_ACTION_1,
+    /* 1 */ EN_STONE_ACTION_2,
+    /* 2 */ EN_STONE_ACTION_CHECK_ITEM,
+    /* 3 */ EN_STONE_ACTION_DRINK_BOTTLE,
+    /* 4 */ EN_STONE_ACTION_5
```

Comment:
```suggestion
    /* 0 */ EN_STONE_ACTION_0,
    /* 1 */ EN_STONE_ACTION_1,
    /* 2 */ EN_STONE_ACTION_CHECK_ITEM,
    /* 3 */ EN_STONE_ACTION_DRINK_BOTTLE,
    /* 4 */ EN_STONE_ACTION_4
```
Best to sync these up with the enum values